src/extension.ts

```diff
@@ -115,6 +118,28 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
     vscode.window.showErrorMessage('Command not available: ${tektonExplorer.reason}');
   }
 
+  vscode.workspace.onDidSaveTextDocument(async (document: vscode.TextDocument) => {
+    let value: string;
+    if (document.uri.authority !== 'loadtektonresource') {
+      const verifyTknYaml = tektonYaml.isTektonYaml(document);
+      if (!context.globalState.get(document.uri.fsPath)) {
```

Comment:
I think using `globalState` in this case is ambiguous, better to use `workspaceState`, as file that user edits is belonged to some vscode workspace.